modules/builtin/src/backend/index.ts

```diff
@@ -14,10 +22,18 @@ const botTemplates: sdk.BotTemplate[] = [
   { id: 'empty-bot', name: 'Empty Bot', desc: `Start fresh with a clean flow` }
 ]
 
+const prompts = [PromptConfirm, PromptDate, PromptNumber, PromptString, PromptEnum, PromptPattern]
+
+const onServerStarted = async (bp: typeof sdk) => {
+  await setupMiddleware(bp, prompts)
+}
+
 const entryPoint: sdk.ModuleEntryPoint = {
-  variables: [BoxedDate, BoxedBoolean, BoxedNumber, BoxedString],
+  onServerStarted,
+  variables: [BoxedDate, BoxedBoolean, BoxedNumber, BoxedString, BoxedEnum],
```

Comment:
Nope, because if you rename your class Boolean, then you can't use the native Boolean method anywhere in your class. And it would be counter-intuitive.